test_apps/test_app/app/contracts/SimpleStorageWithIPFSImport.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.23;
+
+import "https://github.com/status-im/contracts/blob/master/contracts/identity/ERC725.sol";
+import "ipfs://QmcFwDjZov6pD2FXdXtHS6GZrK95tJqKoZjst6gvUbrNmD/contracts/Ownable.sol";
```

Comment:
Don't ipfs uploads end up expiring at some point?

Reply:
Also, you can put that import inside `SimpleStorageWithHttpImport.sol` so we don't have too many contracts.

Reply:
yes no problem.Have you tried to import this .sol file ?

Reply:
No I did not. i'll test everything once the PR is complete.